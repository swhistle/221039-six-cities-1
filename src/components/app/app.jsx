import React from "react";
import {MainScreen} from "../main/main.jsx";

export const App = () => {
  return <MainScreen/>;
};
