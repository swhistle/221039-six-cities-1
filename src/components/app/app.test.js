import React from 'react';
import renderer from 'react-test-renderer';
import {App} from './app.jsx';

it(`App correctly renders`, () => {
  const app = renderer
    .create(<App/>)
    .toJSON();

  expect(app).toMatchSnapshot();
});
