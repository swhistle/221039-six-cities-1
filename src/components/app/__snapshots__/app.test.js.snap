// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders 1`] = `
<div>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                className="header__nav-link header__nav-link--profile"
                href="/login"
                onClick={[Function]}
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                >
                  
                </div>
                <span
                  className="header__login"
                >
                  Sign in
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--property"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="cities tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        >
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item tabs__item--active"
              onClick={[Function]}
            >
              <span>
                Cologne
              </span>
            </a>
          </li>
        </ul>
      </section>
    </div>
    <form
      action="#"
      className="places__sorting"
      method="get"
    >
      <span
        className="places__sorting-caption"
      >
        Sort by
      </span>
      <span
        className="places__sorting-type"
        onClick={[Function]}
        tabIndex="0"
      >
        Popular
        <svg
          className="places__sorting-arrow"
          height="4"
          width="7"
        >
          <use
            xlinkHref="#icon-arrow-select"
          />
        </svg>
      </span>
      <ul
        className="places__options places__options--custom"
      >
        <li
          className="places__option places__option--active"
          onClick={[Function]}
          tabIndex="0"
        >
          Popular
        </li>
        <li
          className="places__option"
          onClick={[Function]}
          tabIndex="0"
        >
          Price: low to high
        </li>
        <li
          className="places__option"
          onClick={[Function]}
          tabIndex="0"
        >
          Price: high to low
        </li>
        <li
          className="places__option"
          onClick={[Function]}
          tabIndex="0"
        >
          Top rated first
        </li>
      </ul>
      <select
        className="places__sorting-type visually-hidden"
        id="places-sorting"
      >
        <option
          className="places__option"
          defaultValue={true}
          value="popular"
        >
          Popular
        </option>
        <option
          className="places__option"
          value="to-high"
        >
          Price: low to high
        </option>
        <option
          className="places__option"
          value="to-low"
        >
          Price: high to low
        </option>
        <option
          className="places__option"
          value="top-rated"
        >
          Top rated first
        </option>
      </select>
    </form>
    <div
      className="cities__places-wrapper"
    >
      <div
        className="cities__places-container container"
      >
        <section
          className="cities__places places"
        >
          <h2
            className="visually-hidden"
          >
            Places
          </h2>
          <b
            className="places__found"
          >
            1
             places to stay in 
            Cologne
          </b>
          <div
            className="cities__places-list places__list tabs__content"
          >
            <article
              className="near-places__card place-card"
            >
              <div
                className="near-places__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                  onMouseOver={[Function]}
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      80
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                      / night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button"
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "100%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/1"
                    onClick={[Function]}
                  >
                    Wood and stone place
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  Private room
                </p>
              </div>
            </article>
          </div>
        </section>
        <div
          className="cities__right-section"
        >
          <section
            className="cities__map map"
          >
            <div
              id="map"
            />
          </section>
        </div>
      </div>
    </div>
  </main>
</div>
`;
